update-extension-dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Placement in root ok? Or somewhere else?

Reply:
That is fine 👍🏻